SUSE/scf
# Istio on Vagrant

## Description

- Brings back https://github.com/SUSE/scf/commit/27486c2dcff7359d5ac4c099aa27d0f19d547711.
- Adds minor changes.
- Adds missing stop.

## Test plan

On a Vagrant machine, run:

- `make istio-run`.
- Check that Istio was installed on k8s.
- `make istio-stop`.
- Check that Istio was uninstalled from k8s.
- Check that `make istio-stop` is idempotent.
- Check that `make istio-run` will perform an upgrade if Istio is already installed.
- `cat ~/.profile` and check for the exported `$PATH` with the Istio bin location.
- Change the version of Istio on `//bin/common/versions.sh` to `1.0.4` and run `make istio-run`. Notice it performs the download of the version and upgrades the installation.
- `cat ~/.profile` and check that `$PATH` with the Istio bin location was updated.
